src/ActionModal/components/DelegationModal.vue

```diff
@@ -251,25 +250,17 @@ export default {
       if (this.isRedelegation) {
         return {
           type: transactionTypes.REDELEGATE,
-          validatorSourceAddress: this.from,
-          validatorDestinationAddress: this.targetValidator.operatorAddress,
-          amount: toMicroUnit(
-            this.amount,
-            this.stakingDenom,
-            this.networks.find(({ id }) => id === this.network)
-          ),
-          denom: toMicroDenom(this.stakingDenom)
+          from: [this.from],
```

Comment:
@bitcoinera here you see that the transactionData already contains all the things that you will need for displaying

Reply:
yes, I'm aware of it 😉

Reply:
O.o oh, I thought we needed toMicroUnit and toMicroDenom to build the transactions! noooo 😨 

I took the time to add a `displayAmount` in my PR 🤦 will need to remove it then

Why did we have it then with the toMicroUnit and MicroDenom???

Reply:
now I just convert all `amount, denom` in the network message creators instead of in the ActionModal. The ActionModal creates values in the TransactionV2 exchange format